AmplifyPlugins/Auth/AWSCognitoAuthPluginTests/ClientBehaviorAPITests/AWSCognitoAuthDeviceBehaviorTests.swift

```diff
@@ -22,10 +25,6 @@ class AWSCognitoAuthDeviceBehaviorTests: XCTestCase {
                          hubEventHandler: MockAuthHubEventBehavior())
     }
 
-    override func tearDown() {
-        Amplify.reset()
-    }
```

Comment:
It seems more logically to do the reset in the `tearDown` method, as a "clean up" step of all tests. Is there a reason to move this to the "setUp"? I would think we can still call `wait(for: 1)` in the tearDown after `Amplify.reset()`